c/src/ml-api-service-db.cc

```diff
@@ -0,0 +1,268 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * @file ml-api-service-db.c
+ * @date 07 Mar 2022
+ * @brief NNStreamer/Service C-API implementation
+ * @see	https://github.com/nnstreamer/nnstreamer
+ * @author MyungJoo Ham <myungjoo.ham@samsung.com>
+ * @author Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <iostream>
+#include <cstring>
+
+#include "ml-api-inference-internal.h"
+#include "ml-service-db.hh"
+#include "ml-api-service.h"
+
+#define ML_DATABASE_PATH      SYS_DB_DIR"/.ml-service-leveldb"
+
+/**
+ * @brief Class for implementation of IMLServiceDB
+ */
+class MLServiceLevelDB : public IMLServiceDB
+{
+public:
+  MLServiceLevelDB (std::string path);
+  virtual ~ MLServiceLevelDB ();
+  virtual void connectDB () override;
+  virtual void setPipelineDescription (std::string name,
```

Comment:
`name` 과 `pipeline_description` 이 함수내에서 변경되지 않는 것 같은데, const 를 붙여서 명확히 해주는게 어떨까요?
```suggestion
  virtual void setPipelineDescription (const std::string name, const std::string pipeline_description)
```

Reply:
네. 그렇게 하는 것이 더 명확해 보이네요.